docs/source/yelpsoa_configs.rst

```diff
@@ -41,21 +41,15 @@ Example::
 All configuration files that define something to launch on a PaaSTA Cluster can
 specify the following options:
 
-  * ``cpus``: Number of CPUs an instance needs. Defaults to 1. CPUs in Mesos
-    are "shares" and represent a minimal amount of a CPU to share with a task
-    relative to the other tasks on a host.  A task can burst to use any
-    available free CPU, but is guaranteed to get the CPU shares specified.  For
-    a more detailed read on how this works in practice, see the docs on `isolation <isolation.html>`_.
+  * ``cpus``: Number of CPUs an instance needs. Defaults to 1. The CPU resource in kubernetes is measured in *CPU* units. One CPU in Kubernetes is equivalent to: 1 AWS vCPU, 1 GCP Core, 1 Azure vCore, 1 Hyperthread on a bare-metal Intel processor with Hyperthreading.
```

Comment:
imo, we can leave this as the previous description + `s/Mesos/Kubernetes` and it'd still be mostly correct :)

the part that would no longer be correct is `A task can burst to use any available free CPU`, since we only allow a limited amount of burst (specified by cpu_burst_add) - but that's fine to leave out since the next explainer is cpu_burst_add and that mentions that the amount of burst is limited in reality :)